ui/app/services/pagination.js

```diff
@@ -83,7 +73,7 @@ export default class StoreService extends Store {
     const skipCache = query.skipCache;
     // We don't want skipCache to be part of the actual query key, so remove it
     delete query.skipCache;
-    const adapter = this.adapterFor(modelType);
+    const adapter = this.store.adapterFor(modelType);
```

Comment:
I tried to leave this file alone as much as possible because `lazyPaginatedQuery` is so widely used and really just updated `this` to `this.store`. I'm sure there's lots of cleanup that could be done here but I didn't want to scope creep too much